e2e-tests/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "aleph-e2e-client"
+version = "0.1.0"
+edition = "2018"
+
+# client
+[dependencies.substrate-api-client]
+git = "https://github.com/scs/substrate-api-client"
+rev = "f9d3a773bb5c76a0c0689b012fc08599beb90ab7"
```

Comment:
This lib stubbornly follows the head of substrate repo (aka master branch), it's so flaky I'd rather fix it to a version that I just know is working.

Reply:
Can you at least replace a commit with a branch or a tag, so that we have some reasonable name?

Reply:
they stopped tagging in may, last tag:
```
error: no matching package named `frame-metadata` found
location searched: https://github.com/paritytech/substrate.git?branch=master
required by package `substrate-api-client v0.6.0 (https://github.com/scs/substrate-api-client?tag=v0.6.0#3c8c04cd)`
    ... which satisfies git dependency `substrate-api-client` of package `aleph-e2e-client v0.1.0 (/aleph-node/e2e-tests)`
```

This is because party has split this package since from the main repo. All branches are feature branches under active development: https://github.com/scs/substrate-api-client/branches/all